test/test.js

```diff
@@ -127,80 +129,34 @@ describe('appmetrics.js', function() {
   });
 
   describe('sendToAnalytics()', function() {
-    it('sends default request', function(done) {
-      const observer = new PerformanceObserver(list => {
-        observer.disconnect();
-
-        const entries = list.getEntries().filter(entry => {
-          return isAnalyticsRequest(entry) &&
-                 entry.name.includes(Math.round(metric.duration)) &&
-                 entry.name.includes(metric.name) &&
-                 entry.name.includes('category_name');
-        });
-
-        assert.equal(entries.length, 1, 'single timing entry');
-        done();
-      });
-      observer.observe({entryTypes: ['resource']});
-
+    let spy;
+    beforeEach(() => {
+      spy = sinon.spy(window, 'ga');
+    });
+    afterEach(() => {
+      spy.restore();
+    });
+    it('sends default request', function() {
       metric.sendToAnalytics('category_name');
+      assert(spy.calledWith('send', 'timing', 'category_name', metric.name, Math.round(metric.duration)));
     });
 
-    it('can override duration and name', function(done) {
-      const observer = new PerformanceObserver(list => {
-        observer.disconnect();
-
-        const entries = list.getEntries().filter(entry => {
-          return isAnalyticsRequest(entry) &&
-                 entry.name.includes('1234567890') &&
-                 entry.name.includes('category_name') &&
-                 entry.name.includes('metric_name');
-        });
-
-        assert.equal(entries.length, 1, 'one timing entry');
-        done();
-      });
-      observer.observe({entryTypes: ['resource']});
-
+    it('can override duration and name', function() {
       metric.sendToAnalytics('category_name', 'metric_name', 1234567890);
+      assert(spy.calledWith('send', 'timing', 'category_name', 'metric_name', 1234567890));
```

Comment:
These are testing that the GA api call is made, not that the entries are properly recorded in the `PerformanceObserver`. Can we keep the tests as is? Think that means removing sinon....?

Reply:
I tried leaving it as it was. However, when I first ran the tests they always failed. They were sending the wrong payload, metric.name was always `google-analytics.com/analytics.js` (or some variation of the GA script address).  

The tests were previously more like integration tests and covered part of the internals of GA, which I'm not really sure is necessary. Since GA is a 'soft' peer dependency, as long as GA behaves then we shouldn't care except that we call the interface correctly.